### ISSUE TITLE
Consider using a different version number to avoid false-positive vulnerability check with old CVEs

### ISSUE THREAD

**Issue by haijian-ccc** [vaadin/open#5]
I see vaadin:open is released directly from version 8 and is now part of vaadin 22.2.11, as a dependency of the license-checker:1.11.2.

The version number 8 could fail the vulnerability check false positively with CVEs for Vaadin 8, e.g. [CVE-2019-25028](https://nvd.nist.gov/vuln/detail/CVE-2019-25028), [CVE-2021-33609](https://nvd.nist.gov/vuln/detail/CVE-2021-33609)

The consequence is that for any Vaadin 22.2.x application with vulnerability check, developers will have to spend some time to figure out why the build suddenly fails and suppress the CVEs, and rebuild, could easily take few hours.

**Comment by manolo** [comment]
@haijian-ccc I guess you meant `23.2.11`.
What is the tool you mainly use for this detection, could it be based on the `dependency-check-maven`? 
Didn't you have other false positives with other vaadin platform modules like license-checker, that also are in old CVE ranges?

**Comment by haijian-ccc** [comment]
Hi @manolo, yes, I meant **23.2.11**, sorry, it was typo.
For the tool, I think we use [org.owasp:dependency-check-core](https://mvnrepository.com/artifact/org.owasp/dependency-check-core).
Based on the report, I only see two CVEs failing [CVE-2019-25028](https://nvd.nist.gov/vuln/detail/CVE-2019-25028), [CVE-2021-33609](https://nvd.nist.gov/vuln/detail/CVE-2021-33609), also I see that `com.vaadin:open` is part of the `licence-checker`?
![image](https://user-images.githubusercontent.com/101243409/207774064-d2c64cf1-fa91-4715-b63f-826f888a054e.png)

For me, after I exclude the `com.vaadin:open` dependency, then it passes the dependency check.

Also `com.vaadin:license-checker` has version **1.11.2**, the two linked CVEs has the affected version range of 7 to 8
![image](https://user-images.githubusercontent.com/101243409/207774665-6cf32356-2427-4217-8305-1b8d0dfd2623.png)

If you have other CVEs you'd like to check, you can send me the numbers and I can check if we have suppressed those.

**Comment by manolo** [comment]
But don't you have also license-checker in the exclusion list, because for me it also reports a false positive 
```
license-checker-1.11.2.jar (pkg:maven/com.vaadin/license-checker@1.11.2, cpe:2.3:a:vaadin:vaadin:1.11.2:*:*:*:*:*:*:*) : CVE-2011-0509
```

**Comment by haijian-ccc** [comment]
Just checked, CVE-2011-0509 seems to has been [suppressed ](https://jeremylong.github.io/DependencyCheck/general/suppression.html)by us on a company level.

**Comment by ZheSun88** [comment]
I have created the FP reports for [DependencyCheck project](https://github.com/jeremylong/DependencyCheck)
there will be new rules to suppress this. 

https://github.com/jeremylong/DependencyCheck/issues/5153
https://github.com/jeremylong/DependencyCheck/issues/5155

**Comment by ZheSun88** [comment]
the fix for has been merged to the plugin for dependency checking and it will be shipped with version 7.4.2 in the future. 

if it is possible, you can test with its 7.4.2-SNAPSHOT. 👍

**Comment by Artur-** [comment]
Sounds like this was fixed a long time ago
